__init__.py

```diff
@@ -16,41 +16,37 @@
 # along with Mycroft Core.  If not, see <http://www.gnu.org/licenses/>.
 
 from adapt.intent import IntentBuilder
-
-from mycroft.skills.core import MycroftSkill
-from mycroft.util.log import getLogger
-
-__author__ = 'eward'
-
-LOGGER = getLogger(__name__)
-
+from mycroft import MycroftSkill, intent_file_handler, intent_handler
 
 class HelloWorldSkill(MycroftSkill):
     def __init__(self):
-        super(HelloWorldSkill, self).__init__(name="HelloWorldSkill")
+        MycroftSkill.__init__(self)
```

Comment:
We can skip the entire `__init__` since it doesn't do it or use `super().__init__()`

Reply:
Kept it and declared a random variable, but switched to super